staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -263,3 +355,43 @@ type CustomResourceSubresourceScale struct {
 	// +optional
 	LabelSelectorPath *string `json:"labelSelectorPath,omitempty" protobuf:"bytes,3,opt,name=labelSelectorPath"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConversionReview describes a conversion request/response.
+type ConversionReview struct {
```

Comment:
Should this be in the same api group? I expected it in a different api group since that's how the existing webhooks seem to work. I'm not sure we have a great reason for doing it that way-- maybe @liggitt knows.

Reply:
This is my only question on the PR I think.

Reply:
Putting the webhook types in this group means that someone's conversion webhook would have to depend on `k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1` to get the versioned struct, which seems slightly odd.

If these types make sense for conversion in general, even apart from custom resources, I could see putting them in `k8s.io/api/...`. Not sure which group.

@deads2k @sttts, do you remember more of the context around placement of the admission types?

Reply:
I think something like `k8s.io/api/conversion/v1beta1` make sense. AdmissionReview is in `k8s.io/api/admission/v1beta1`.

Reply:
> @deads2k @sttts, do you remember more of the context around placement of the admission types?

Honestly, I don't remember an explicit choice.  I would choose to place them in the genericapiserver since that's what I would choose to build my conversion webhook from and we know that that kube-apiserver isn't going to serve these, but I don't feel super strongly.

Reply:
I don't think we would use conversion webhooks for anything other than CRDs (a point agains `k8s.io/api/conversion/v1beta1`).

Concerning imports, we have a [sample conversion webhook](https://github.com/kubernetes/kubernetes/pull/67006/commits/b31503af5365263544ad36e73abbd268085d65c5) we can look at 

The imports there are apimachinery and apiextensions-apiserver. Between those `apimachinery` is to generic in my opinion and `apiextensions-apiserver` is a far better option. I think we should keep the location as is.

Reply:
I don't think it belongs in apimachinery. Between api, apiserver, and extensions-apiserver I don't feel strongly.  I do think that I would keep the CRD API local to the extensions-apiserver though.  Generated clients are composeable, so we don't need to bundle them all together and we haven't had significant trouble related to the two packages so far.

Reply:
keeping them in the `apiextensions.k8s.io` group seems ok

Reply:
+1 for apiextensions-apiserver, if we don't have a strong reason for some other place.